pkgs/by-name/sc/scienceplots/package.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, python3Packages
+, fetchPypi
+}:
+
+python3Packages.buildPythonPackage rec {
```

Comment:
should be in pkgs/development/python-modules

Reply:
Are you sure? There I though new packages should be put into `pkgs/by-name/`. There are also other python packages in this path

Reply:
If you put them in python-modules, they will be available for all Python versions. In by-name they will only be available for one version.